test/test_ssl.rb

```diff
@@ -1222,6 +1222,56 @@ def test_get_ephemeral_key
     end
   end
 
+  def test_fallback_scsv
```

Comment:
This test case is failing with LibreSSL or older patch versions of OpenSSL 1.0.1 because they lack support for TLS_FALLBACK_SCSV. `pend` if the method does not exist.

https://travis-ci.org/ruby/openssl/jobs/294781400#L660

Reply:
Fixed